src/graphql/fragments/actions.graphql

```diff
@@ -23,8 +23,39 @@ fragment ColonyAction on ColonyAction {
     ...Domain
   }
   createdAt
-  isMotion
   colony {
     colonyAddress: id
   }
+  isMotion
+  motionData {
+    ...MotionData
+  }
+}
+
+fragment MotionData on MotionData {
+  motionDomainId
+  motionStakes {
+    raw {
+      yay
+      nay
+    }
+    percentage {
+      yay
+      nay
+    }
+  }
+  usersStakes {
+    address
+    stakes {
+      raw {
+        yay
+        nay
+      }
+    }
+  }
```

Comment:
These are so confusing. Yeah, after a while I understand what's going on, but just reading them outright is gonna cause trouble for sure.

Problem is I don't have a better naming alternative, without going super descriptive